openedx/core/djangoapps/xblock/app_impl/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+Interface that defines implementation differences between the LMS and Studio
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from django.conf import settings
+
+if settings.ROOT_URLCONF == 'cms.urls':  # We are in Studio:
+    from . import studio_impl as parent_app
+else:
+    from . import lms_impl as parent_app
```

Comment:
Is it reasonable to do this with AppConfig -- make a separate configs for Studio vs. LMS for them to put in their respective INSTALLED_APPS?